.github/dependabot.yml

```diff
@@ -1,7 +1,12 @@
 version: 2
 updates:
-- package-ecosystem: pip
-  directory: "/"
-  schedule:
-    interval: daily
-  open-pull-requests-limit: 10
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
Is daily really needed?

```suggestion
      interval: weekly
```

Reply:
We have daily for the actions on most the libraries.